packages/types/src/appearance.ts

```diff
@@ -590,6 +590,11 @@ export type Layout = {
    * prebuilt components. Clicking the link will open the passed URL in a new tab
    */
   privacyPageUrl?: string;
+  /**
+   * This option enables the shimmer animation for the avatars of <UserButton/> and <OrganizationSwitcher/>
```

Comment:
💯 
Thanks for writing the docs here, we should strive to move everything in line